src/style.css

```diff
@@ -1,3 +1,74 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+  width: 100%;
+  padding: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  margin: 0;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  margin-right: 20px;
+  padding: 0;
+}
+
+a {
```

Comment:
Don't style tags, use classes instead of it. Fix all cases

Reply:
Thank you for your suggestion! Are you mean about html, body and header too?